### ISSUE TITLE
Outline/Discussion of Goals

### ISSUE THREAD

**Issue by thequbit** [mediapublic/mediapublic#24]
I am reposting this here, so it doesn't get lost at the bottom of another issue thread ...

---

@GabeIsman wrote:
Maybe this is the wrong place to bring up these concerns but I continue to have nagging doubts that we really haven't yet focused in on what exactly we are trying to do, and what assumptions we are testing.

I see three distinct problems that we are trying to solve.
1. The public has no way of contributing to a public radio station in non-monetary ways.
2. Public radio stations don't communicate very well with each other.
3. It's hard to find the excellent content that public radio stations have produced in the past.

I think we could take any one of these three problems and build something really awesome to tackle it over the next six months. I have this terrible feeling that if we try to take on all three we're going to end up sort of half-assing all of them.

The ways we've talked about attempting to solve each of these problems are respectively (forgive my simplification):
1. Adapt Midas to work really well for public radio stations. Integrate it into their CRM's so memberships can be awarded, promote a culture of involving the public within radio stations, maybe we are also making Hive easily deployable by a station on their own archives.
2. Create a sort of social network for public radio, where each station has a page, each journalist/producer/coder/etc has a page, they can all share information with each other through how-to's and open-source projects.
3. Create a page where archival content can be resurfaced, where people can make and share playlists, where people can add audio (scraped by us?), where audio will be surfaced in interesting using interesting tags, we would also build the tagging system?

Each of these projects, frankly, sounds like an ambitious one for a small handful of people who all have other jobs to tackle in six months. Trying to do them all at once sounds impossible.

I propose that we think a little harder about which of these three projects is most important, most feasible, and most effectively tests our assumptions. I propose that we pick one, and build it, launch it, see what happens, and then decide what to do next.

We can always keep building iteratively. If we adapt Midas and find that what's really lacking is the communication between stations, we build that next. If every station wants us to make crowd-sourced archival work easier, we build out a Hive integration next.

I hope this doesn't come off as overly critical or too harsh, I just keep having this bad feeling that we haven't really honed in on what it is that we really want to get out of this thing. I would much rather focus on something and build something small really well that can become part of something larger down the line, than build something big and not very developed.

My two cents. I look forward to hearing everyone else's. Am I crazy? Is this all actually easier than I am imagining?

**Comment by thequbit** [comment]
If I had my vote, I would say number 2, and then try and pull in features of  number 1 as we go along.  Something like ...
1. Create Station Landing Pages
2. Create Ability to Search Stations
3. Create Learn Pages
4. Create Ability to Search for Howtos ( Learn Pages )
5. Create Person Pages
   - Station Personal
   - Contributors ( Coders, Translators, Taggers, etc. )
   - Everyone else ( general users )
6. Create Ability to Search for Person Pages
7. Create Task Page ( "I need help with ..." )
8. Create Ability to Search for Task Pages
9. Add some kind of badge/kick-back system
   - This is for folks who contribute to a station by "solving" a task

I'm open for all other suggestions thought!

**Comment by melodykramer** [comment]
I really like what @thequbit suggests. I think there's a big need for communication, both with stations and the public. I'm hopeful that we will be able to apply for / receive additional funding, and having a prototype, a user base, and a roadmap will be key to future funding, I think.

**Comment by melodykramer** [comment]
Notes from MVP chat [here](https://github.com/mediapublic/mediapublic/wiki/Notes-for-Station-Page-Experience.)

**Comment by melodykramer** [comment]
I keep thinking about this and want to push us a bit on the stations as landing pages idea. 

[This story](http://michiganradio.org/post/unique-ann-arbor-editor-prefers-print-journalism#stream/0) is interesting no matter where you live. I don't live in Michigan. How do I find that story?

People who move to Rochester don't know the station's call letters are WXXI. They know that they might want to listen to public radio, in Rochester - or maybe they want to listen to [this story](http://michiganradio.org/post/unique-ann-arbor-editor-prefers-print-journalism#stream/0)

**Comment by nkkl** [comment]
I think stations as landing pages is step 1. I think we evolve from there,
but we need to be thoughtful about that though, because it's harder to do
well.

@nkkl
323.205.6533

**Comment by thequbit** [comment]
First, @melodykramer I modified your comment because the period in the URL to the wiki wasn't being included by github "auto make it a link".  It is linking correctly now.

Second, @melodykramer I was thinking that we would still have a "search" page.  A page that would look similar to the craigs list "browse for a city" page ( ref: http://www.craigslist.org/about/sites ).  We can even do the 'share your location' thing like craigslist does ( I just did this on yellr.net, and it's super awesome/easy to do ).

I think the core of the site ( from what I've gathered ) is still the 'station landing page'.  The question becomes, how do we make sure folks are able to easily get content when there are almost 1000 pages generating that content ...
